src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -169,6 +169,11 @@ def test_burst_pattern_pass_fail(multi_instrument_session):
 
 
 def test_source_waveform_parallel_broadcast(multi_instrument_session):
+    '''Test methods for parallel broadcast.
```

Comment:
Test methods for using source waveform with parallel sourcing and broadcast data mapping.

Reply:
Updated.